src/core_read_parameter_file.c

```diff
@@ -234,7 +272,7 @@ int read_parameter_file(const int ThisTask, const char *fname, struct params *ru
                     *((double *) ParamAddr[j]) = atof(buf2);
                     break;
                 case STRING:
-                    strcpy(ParamAddr[j], buf2);
+                    strncpy(ParamAddr[j], buf2, MAX_STRING_LEN - 1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Easily used incorrectly; doesn't always \0-terminate or check for invalid pointers [MS-banned] (CWE-120).  ](https://app.codacy.com/manual/jacobseiler/sage-model/pullRequest?prid=4620074)